internal/domain/pigeomail/telegram/user_input.go

```diff
@@ -16,7 +18,10 @@ func (b *Bot) handleUserInput(update *tgbotapi.Update) {
 	var state pigeomail.UserState
 	var err error
 	if state, err = b.svc.GetUserState(ctx, update.Message.Chat.ID); err != nil {
-		b.handleError(err, update.Message.Chat.ID)
+		if err != customerrors.ErrNotFound {
+			sentry.CaptureException(err)
+			b.handleError(err, update.Message.Chat.ID)
```

Comment:
move sentry call to handleError method (to capture unexpected errors)
and add block before handleError call here (to check that we can ignore user input)
if errors.Is(err, customerrors.ErrNotFound) {
          return
}